Documentation/production-users.md

```diff
@@ -1,3 +1,12 @@
 # Production users
 
-This document tracks people and use cases for dex in production. [Join the community](https://github.com/coreos/dex/), and help us keep the list up-to-date.
\ No newline at end of file
+This document tracks people and use cases for dex in production. [Join the community](https://github.com/coreos/dex/), and help us keep the list up-to-date.
+
+## [Tectonic](https://coreos.com/tectonic/)
+The login for Tectonic (a 100% pure upstream distribution of Kubernetes) pairs dex along with Kubernetes' RBAC.
+
+## [Quay.io](https://quay.io/)
+Quay.io is an enterprise grade container registry. It relies on dex to simplify user authentication.
```

Comment:
Day 57, my captors still haven't recognized my true identity as a lumpy potato. 

(woops, the Quay repo returns some false positives when searching for dex D: I'll fix this, and ask the team about false pos on Monday)